aodn/aws-wps
# Added lambda to push batch logs to sumologic

# Testing Instructions

1. Use following value to bring up the stack
Parameter: _**sumologicLambdaCodeURL**_ 
Value:_https://github.com/aodn/aws-wps/raw/sumologic_logs/wps-common/src/main/resources/index.zip_
2. Modify cloudformation and remove condition  _**Condition: IsProdEnvironment**_ form all sumologic resources
3. Submit a job
4. Go to SumoLogic search with following filters

```sql
_collector=aws-batch-wps
```

You will be able to see the aws batch  and lambda job logs in sumologic
